gtk4/src/shortcut_trigger.rs

```diff
@@ -3,6 +3,43 @@
 use crate::ShortcutTrigger;
 use glib::translate::*;
 use glib::IsA;
+use std::fmt;
+
+impl fmt::Debug for ShortcutTrigger {
+    #[doc(alias = "gtk_shortcut_trigger_print")]
+    fn print(&self) -> Option<String> {
+        unsafe {
+            let mut string = String::uninitialized();
+            let ret = ffi::gtk_shortcut_trigger_print(
+                self.as_ref().to_glib_none().0,
+                string.to_glib_none().0,
+            );
+            if from_glib(ret) {
+                Some(string)
+            } else {
+                None
+            }
+        }
+    }
```

Comment:
This can't work, you need to implement the fmt function. Which would call the print function.